src/editors/containers/VideoEditor/components/VideoSettingsModal/components/ThumbnailWidget/index.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+  intlShape,
+} from '@edx/frontend-platform/i18n';
+import {
+  Image,
+  Stack,
+  Button,
+  OverlayTrigger,
+  Icon,
+  IconButton,
+  Tooltip,
+  Alert,
+} from '@edx/paragon';
+import { Delete, FileUpload } from '@edx/paragon/icons';
+
+import { actions, selectors } from '../../../../../../data/redux';
+import { acceptedImgKeys } from './utils';
+import * as hooks from './hooks';
+import messages from './messages';
+
+import CollapsibleFormWidget from '../CollapsibleFormWidget';
+import FileInput from '../../../../../../sharedComponents/FileInput';
+
+/**
+ * Collapsible Form widget controlling video thumbnail
+ */
+export const ThumbnailWidget = ({
+  // injected
+  intl,
+  // redux
+  thumbnail,
+  updateField,
+}) => {
+  const [thumbnailSrc, setThumbnailSrc] = React.useState(thumbnail);
+  const fileInput = hooks.fileInput({ setThumbnailSrc });
+
+  return (
+    <CollapsibleFormWidget title="Thumbnail">
+      {thumbnail ? (
+        <Stack direction="horizontal" gap={3}>
+          <Image src={thumbnailSrc} alt={intl.formatMessage(messages.thumbnailAltText)} />
+          <OverlayTrigger
+            key="top"
+            placement="top"
+            overlay={(
+              <Tooltip>
+                <FormattedMessage {...messages.deleteThumbnail} />
+              </Tooltip>
+            )}
+          >
+            <IconButton
+              className="d-inline-block"
+              iconAs={Icon}
+              src={Delete}
+              onClick={() => updateField({ thumbnail: null })}
```

Comment:
we don't do anything on the backend at this point?

Reply:
I am not storing the thumbnail on the backend until all the settings are saved, so there is nothing to delete in the backend. So I only need to update the redux store.